app/src/main/java/com/work/restaurant/data/source/local/eat/EatLocalDataSource.kt

```diff
@@ -5,28 +5,36 @@ import com.work.restaurant.network.room.entity.EatEntity
 interface EatLocalDataSource {
 
     fun addEat(
+        userId: String,
         date: String,
         time: String,
         type: Int,
         memo: String,
-        callback: EatLocalDataSourceCallback.AddEatCallback
+        callback: (Boolean) -> Unit
```

Comment:
```suggestion
        callback: (isSuccess: Boolean) -> Unit
```
만든건 잘 하셨는데 네이밍을 달아주는게 보통의 경우에는 좀 더 좋은 것 같아요. 
Boolean형 만으로는 이 변수의 의미가 정확히 뭔지 파악이 안되니까요.

Reply:
278fe4f 반영하였습니다.